ql/src/java/org/apache/hadoop/hive/ql/parse/repl/metric/MetricSink.java

```diff
@@ -121,6 +121,7 @@ public void run() {
             ObjectMapper mapper = new ObjectMapper();
```

Comment:
is this method supposed to be fast? (because `ArrayList` was created for a specified size)
...anyway try not to throw away `ObjectMapper` instances right after use - `ObjectMapper`'s first time use cost could be high